modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -411,36 +433,8 @@ test('Can sort Data Set by different columns', async ({
 			rowIndex: 3,
 		});
 
-		dataSetsPage.sortBy('Name');
-		await assertTableCellContent({
-			dataSetConfig: skusDataSetConfig,
-			page,
-			rowIndex: 0,
-		});
-		await assertTableCellContent({
-			dataSetConfig: productsDataSetConfig,
-			page,
-			rowIndex: 1,
-		});
-		await assertTableCellContent({
-			dataSetConfig: catalogsDataSetConfig,
-			page,
-			rowIndex: 2,
-		});
-		await assertTableCellContent({
-			dataSetConfig: blogPostsDataSetConfig,
-			page,
-			rowIndex: 3,
-		});
-	});
+		await dataSetsPage.sortBy('Name');
 
-	await test.step('Check that it is possible to sort Data Sets by REST Application', async () => {
```

Comment:
I removed this step. It was failing because in the test we are using three of the same REST Applications. Sorting them returns a random order in response. Other columns have all different values. I think other columns are more then enough to assert that sorting works, so deleted step as the simplest solution. If we do want it, we can refactor samples in a followup. @juanjofgliferay

Reply:
It's fine for me removing this step right now, but better than refactoring the samples we should find a way to always ensure the order, like using a second argument for the sort if possible.
Thinking in the Users MVP, where columns like name or job title could have the same content, not being able to ensure the order seems a bug to me.

Reply:
I wouldn't call it a bug, but you are right, it's a behavior that could be improved. Besides the explicit option to select secondary sort, there are some smart behaviors like in Excel that remembers last sort and sets that as secondary. Or we could be opinionated and set some arbitrary column like `name` as secondary. Or, we could let AI decide :slightly_smiling_face: 

It's a product call cc @herrardo  For functionality we have now, I believe the only thing that's important is primary sort. And if there are duplicates of items in the sorting column, it is fine that their order is random.